test/Microsoft.Azure.SignalR.E2ETests/Management/ServiceHubContextE2EFacts.cs

```diff
@@ -31,9 +32,11 @@ public class ServiceHubContextE2EFacts : VerifiableLoggedTest
         private static readonly string[] _groupNames = GetTestStringList("Group", GroupCount);
         private static readonly ServiceTransportType[] _serviceTransportType = new ServiceTransportType[] { ServiceTransportType.Transient, ServiceTransportType.Persistent };
         private static readonly string[] _appNames = new string[] { "appName", "", null };
+        private readonly ITestServerFactory _testServerFactory;
```

Comment:
How about make it static and constrcut here?

Reply:
I think it seems not the field should be static